models/evc.py

```diff
@@ -399,6 +419,36 @@ def archive(self):
         """Archive this EVC on deletion."""
         self.archived = True
 
+    def _use_uni_vlan(self, uni: UNI):
+        """Use tags from UNI"""
+        if uni.user_tag is None:
+            return
+        tag = uni.user_tag.value
+        tag_type = uni.user_tag.tag_type
+        if isinstance(tag, int):
+            if not uni.interface.use_tags([tag, tag], tag_type):
+                intf = uni.interface.id
+                raise ValueError(f"Tag {tag} is not available in {intf}")
+            uni.interface.notify_interface_tags(self._controller)
+
+    def make_uni_vlan_available(self, uni: UNI):
+        """Make available tag from UNI"""
+        if uni.user_tag is None:
+            return
+        tag = uni.user_tag.value
+        tag_type = uni.user_tag.tag_type
+        if isinstance(tag, int):
+            if not uni.interface.make_tags_available([tag, tag], tag_type):
+                intf = uni.interface.id
+                log.warning(f"Tag {tag} was already available in {intf}")
+            else:
+                uni.interface.notify_interface_tags(self._controller)
```

Comment:
`interface.notify_interface_tags` should be only called internally in the core, in fact this could be renamed as `interface._notify_interface_tags` with the underscore to make it clear that's not supposed to be called by NApps. If you could refactor this part, and ensure that on your Kytos PR when `make_tags_available` succeeds then it calls internally the notification. The core should be abstracting this part facilitating for clients too.

Reply:
Deleted direct calling of `notify_interface_tags()`

Reply:
Excellent